### ISSUE TITLE
Can admin see the accepted users list somewhere?

### ISSUE THREAD

**Issue by sumitt** [keensoft/alfresco-agreement-filter#3]
Hello,

I have installed this add-on successfully on alfresco community 5.2 for ubuntu and it's working fine. Now another use case for me is that admin could check and find a list somewhere for the accepted users. is this happening currently by this add-on or any possibility to make this? Looking for your kind help.

Thanks
Sumit

**Comment by angelborroy** [comment]
There is no option in Share web UI to get this information, but you can query the repository by using following property:

https://github.com/keensoft/alfresco-agreement-filter/blob/master/agreement-filter-repo/src/main/java/es/keensoft/alfresco/action/webscript/AgreementPost.java#L35

Every user accepting the agreement have "agreementChecked" property set to true.

**Comment by sumitt** [comment]
Thanks angelborroy for your kind help.

I have tried this in a script under Data Dictionary/Scripts. I was able to find other user properties like first name, last name and user name but agreementChecked is giving null for a true value. below is my script:

var user;
var nodes = people.getPeople(null);
for each(var node in nodes)
{
   user = utils.getNodeFromString(node); 
   logger.log(user.properties["cm:userName"] + " '" + user.properties["cm:firstName"] + "' '" + user.properties["cm:lastName"] + "'" + user.properties["agreementChecked"]);
}
other properties has some prefix like cm but this is not. Above script printing "null" for "agreementChecked" while this is true for that user seen in admin console.

**Comment by angelborroy** [comment]
Did you try "cm:agreementChecked"?

**Comment by sumitt** [comment]
Yes, I tried but no effect. Below I am adding partial output of user.properties which shows agreementChecked. but I am unable to fetch it's value.

 {http://www.alfresco.org/model/content/1.0}firstName=Sumit, {}agreementChecked=true, {http://www.alfresco.org/model/system/1.0}node-uuid=8ecc8664-8b74-46c7-bd72-eacc4de79d15, {http://www.alfresco.org/model/system/1.0}cascadeCRC=1749806651}

If I log user.properties["cm:agreementChecked"] then it's blank. And if I add this in csv then csv show it's value null.

Thanks.

**Comment by sumitt** [comment]
Hi angelborroy, 

I am waiting for your help.

Thanks
Sumit

**Comment by angelborroy** [comment]
You can try getting the property with full QName.

Something like this:

`user.properties["{}agreementChecked"]`

**Comment by sumitt** [comment]
Thanks you very much angelborroy. user.properties["{}agreementChecked"] worked.

Now I working on the report generation part for accepted users.

-Sumit
